compiler/src/iree/compiler/Codegen/LLVMGPU/LLVMGPUDistribute.cpp

```diff
@@ -41,10 +43,11 @@ struct LLVMGPUDistributePass
 
     IRRewriter rewriter(funcOp->getContext());
     rewriter.setInsertionPoint(funcOp);
-    auto walkResult = mlir::linalg::rewriteMapNestedForeachThreadToGpuThreads(
-        rewriter, funcOp, workgroupSize, false);
+    DiagnosedSilenceableFailure const result =
+        mlir::transform::gpu::mapNestedForeachToThreadsImp(
```

Comment:
This should be called "Impl" not "Imp"

Please update upstream and cherry-pick again, you can commit this directly upstream without review once tests pass locally.